.github/workflows/synopsys-schedule.yaml

```diff
@@ -19,11 +19,11 @@ jobs:
       - name: Build Project
         run: make build
 
-      - name: Run Synopsys Detect
-        uses: synopsys-sig/detect-action@v0.3.4
+      - name: Black Duck Full Scan
+        uses: synopsys-sig/synopsys-action@v1.10.0
```

Comment:
Can we use latest tag `v1.13.1`?

Reply:
interesting , i'm curious why dependabot don't catch this one too